dev/docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+service:
+  extends:
+    file: ../common.yml
+    service: service
```

Comment:
this should get the old volume too: ``./payments-service/:/srv/payments-service`